io/disk/iozone.py

```diff
@@ -0,0 +1,649 @@
+import os
```

Comment:
As you port this from autotest, you should mention the original owner and the same/compatible license on top of this file. (see https://github.com/avocado-framework/avocado/blob/master/avocado/utils/software_manager.py as example)